From e225969f90432daf29eff1dde7da573c8183e6f5 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Mon, 27 Sep 2021 15:11:19 +0200
Subject: [PATCH 057/529] ARM: dts: stm32mp13: fix compatible for BSEC

Use the new compatible for stm32mp13 support.

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I167a5efc1f8777cce14518c6fa38400ac684de3e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/272814
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Lionel DEBIEVE <lionel.debieve@foss.st.com>
---
 arch/arm/boot/dts/stm32mp131.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/stm32mp131.dtsi
+++ b/arch/arm/boot/dts/stm32mp131.dtsi
@@ -398,7 +398,7 @@
 		};
 
 		bsec: efuse@5c005000 {
-			compatible = "st,stm32mp15-bsec";
+			compatible = "st,stm32mp13-bsec";
 			reg = <0x5c005000 0x400>;
 			#address-cells = <1>;
 			#size-cells = <1>;
